internal/shell/shell.go

```diff
@@ -101,8 +101,10 @@ func RunInteractiveShell(input string, wait bool, getOutput bool) (string, error
 	c := make(chan os.Signal, 1)
 	signal.Reset(os.Interrupt)
 	signal.Notify(c, os.Interrupt)
-	cmd.Start()
-	err = cmd.Wait()
+	err = cmd.Start()
+	if err == nil {
```

Comment:
Print error if it failed?

I've just tried this, seems to work ok (e.g. when trying to run `RunInteractiveShell()` with a non-existing command):
```
--- a/internal/shell/shell.go
+++ b/internal/shell/shell.go
@@ -104,6 +104,8 @@ func RunInteractiveShell(input string, wait bool, getOutput bool) (string, error
 	err = cmd.Start()
 	if err == nil {
 		err = cmd.Wait()
+	} else {
+		screen.TermMessage(err)
 	}
 
 	output := outputBytes.String()
```

Reply:
I think it can be a bit confusing having to press enter key 2 times if `wait` is `true`, but I will change it so that the error is printed and enter key has to be pressed even if `wait` is `false` when an error occurs in this pull request.

Reply:
Good point. But with your version it prints an extra empty string if `wait` is false. What about:

```
		if (!wait) {
			screen.TermMessage(err)
		} else {
			fmt.Println(err)
		}
```

...Or actually it seems cleaner to move the "main" `screen.TermMessage()` code to the successful branch of the `if`. Then it would be just:

```
	err = cmd.Start()
	if err == nil {
		err = cmd.Wait()
		if wait {
			// This is just so we don't return right away and let the user press enter to return
			screen.TermMessage("")
		}
	} else {
		screen.TermMessage(err)
	}
```